src/main/java/org/broadinstitute/hellbender/tools/exome/ACNVCopyRatioModeller.java

```diff
@@ -362,13 +362,13 @@ private CopyRatioDataCollection(final SegmentedModel segmentedModel) {
                 throw new IllegalArgumentException("Cannot construct CopyRatioDataCollection with no target-coverage data.");
             }
             //construct coverages and number of targets per segment from the segments and Genome in SegmentedModel
-            final TargetCollection<TargetCoverage> targetCoverages = segmentedModel.getGenome().getTargets();
+            final TargetCollection<ReadCountRecord.SingletonRecord> targetCoverages = segmentedModel.getGenome().getTargets();
```

Comment:
Why is this not a ReadCountCollection?  Why use a ReadCountRecord.SingletonRecord?